degiro_connector/trading/actions/action_check_order.py

```diff
@@ -132,14 +132,14 @@ def check_order(
 
         try:
             response_raw = session.send(prepped)
-            response_raw.raise_for_status()
             response_dict = response_raw.json()
```

Comment:
Are you sure this endpoint returns a valid JSON response even when Status Code is not 200?

Reply:
I know that the web UI is using the returned JSON to provide human readable error messages. Examples of such errors are tick size errors or lot size errors. I haven't found a cleaner way of finding the lot- or tick-size for some markets rather than brute force triggering the error and parsing the error messages.